aws/rust-runtime/aws-sigv4/Cargo.toml

```diff
@@ -11,7 +11,6 @@ repository = "https://github.com/awslabs/smithy-rs"
 [features]
 sign-http = ["http", "percent-encoding", "form_urlencoded"]
 sign-eventstream = ["aws-smithy-eventstream", "bytes"]
-default = ["sign-http"]
```

Comment:
I think `aws-sigv4` is another of the odd exceptions since it already had users before the alpha SDK was launched. I don't think there's any harm in keeping `sign-http` default here. This crate is intended to be used standalone from the SDK unlike the other runtime crates.